docs/ops/image/Interpolate_11.md

```diff
@@ -0,0 +1,137 @@
+    * `round_prefer_floor` - this mode is known as round half down.
+    * `round_prefer_ceil` - it is round half up mode.
+    * `floor` - this mode computes the largest integer value not greater than the rounded value.
+    * `ceil` - this mode computes the smallest integer value not less than the rounded value.
+    * `simple` - this mode behaves as `ceil` mode when `Interpolate` is downsample, and as dropping the fractional part otherwise.
+  * **Type**: string
+  * **Default value**: `round_prefer_floor`
+  * **Required**: *no*
+
+* *antialias*
+
+  * **Description**: *antialias* is a flag that specifies whether to perform anti-aliasing.
+  * **Range of values**:
+    * false - do not perform anti-aliasing
+    * true - perform anti-aliasing
+  * **Type**: boolean
+  * **Default value**: false
+  * **Required**: *no*
+
+* *pads_begin*
+
+  * **Description**: *pads_begin* specifies the number of pixels to add to the beginning of the image being interpolated. This addition of pixels is done before the interpolation calculation.
+  * **Range of values**: list of non-negative integer numbers
+  * **Type**: `int[]`
+  * **Default value**: `[0]`
+  * **Required**: *no*
+
+* *pads_end*
+
+  * **Description**: *pads_end* specifies the number of pixels to add to the end of the image being interpolated. This addition of pixels is done before the interpolation calculation.
+  * **Range of values**: list of non-negative integer numbers
+  * **Type**: `int[]`
+  * **Default value**: `[0]`
+  * **Required**: *no*
+
+* *cube_coeff*
+
+  * **Description**: *cube_coeff* specifies the parameter *a* for cubic interpolation (see, e.g.  [article](https://ieeexplore.ieee.org/document/1163711/)).  *cube_coeff* is used only when `mode == cubic`.
+  * **Range of values**: floating-point number
+  * **Type**: any of supported floating-point type
+  * **Default value**: `-0.75`
+  * **Required**: *no*
+
+**Inputs**
+
+*   **1**: `image` - tensor of type *T* with data for interpolation. **Required.**
+
+*   **2**: `scales_or_sizes` - 1D tensor containing the data used to calculate the spatial output shape. The number of elements must match the number of values in the `axes` input tensor, the order needs to match as well. The type of this input tensor is either *T_SCALES* or *T_SIZES* depending on the value of the `shape_calculation_mode` attribute. **Required.**
+
+*   **4**: `axes` - 1D tensor of type *T_AXES* specifying dimension indices where interpolation is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, e.g. `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`. These indices should be non-negative integers from `0` to `rank(image) - 1` inclusively.  Other dimensions do not change. The order of elements in `axes` attribute matters and is mapped directly to the elements in the 2nd input `scales_or_sizes`. **Optional** with default value `[0,...,rank(image) - 1]`.
```

Comment:
If not provided, the default axes should be [ rank(image) - 1 - spatial_rank , ... , rank(image) - 1]? where spatial_rank is elements number in `scales_or_sizes`, as `axes `and `scales_or_size `should have the same number elements.

Reply:
@mitruska , for CHW layout, when scale is {0.5, 0.5}, and default axis is set to [0,1]([0,..., spatial_rank - 1]), which means resize on C and H dimension. 
This is an optional input, the default value just set for most common case(planar layout). If this is not planar layout or other specific case from scratch, we should set it explicitly. For example, nchw input with scale[0.3, 0.8] is for W, H dimension respectively, we need set axis with [3,2].

Reply:
I am not sure, but in my thought, the most common inputs is HW, CHW and NCHW, and set scales only for spatial dimensions(H and W). In this case the default of [rank(image) - 1 - spatial_rank, ... , rank(image) - 1] is the correct default value. I am OK if it's convenient to convert and satisfy "axes and scales_or_size should have the same number elements"

Reply:
I see the point, and I agree it could be more convenient for NCHW layout,
but in my opinion as long as the new default value means more steps during conversion, it shouldn't be changed.